public/css/client.css

```diff
@@ -226,59 +223,111 @@ canvas {
 /*login register */
 #repeatPassword {
     display: none;
+    margin-bottom: 1%;
 }
 
 #username, #password, #repeatPassword, #codeInput, #usernameInput {
     background-image: url("../assets/images/buttons/brownButtonBig.png");
     background-position: center;
     text-align: center;
-    margin: 1%;
-    max-width: 30vw;
+    margin-top: 1%;
+    max-width: 40vw;
     font-size: 2.5vw;
     color: #574741;
-    box-shadow: 5px 5px 1px #574741 inset, -5px -5px 1px #4d3e38 inset;
-    height: 5vw;
+    height: 6vw;
     background-size: cover;
+    border: none;
 }
 
 ::placeholder {
     color: #574741;
 }
 
-#loginButton, #registerButton, #joinGameButtonAFterCodeInput, #playGameButton {
+#loginButton, #registerButton, #joinGameButtonAFterCodeInput {
     background-image: url("../assets/images/buttons/orangeButtonSmall.png");
     background-position: center;
     color: #73340e;
     border-radius: 0;
-    box-shadow: 5px 5px 1px #73340e inset, -5px -5px 2px #58280a inset;
     font-size: 1.5vw;
-    width: 45%;
-    height: 3vw;
+    width: 49%;
+    height: 5vw;
     background-size: cover;
 }
 
 .buttonControl {
-    max-width: 30vw;
+    max-width: 40vw;
     margin: auto;
     text-align: center;
     display: flex;
     justify-content: space-between;
 }
 
-#returnToMainMenuButton, #removePlayerAndGoToMainMenu {
+#returnToMainMenuButton, #removePlayerAndGoToMainMenu, #playGameButton {
     background-image: url("../assets/images/buttons/grayButtonSmall.png");
     background-position: center;
     color: #4c4a49;
-    box-shadow: 5px 5px 1px #4c4a49 inset, -5px -5px 2px #444241 inset;
     font-size: 1.5vw;
     border-radius: 0;
     margin-top: 1%;
     background-size: cover;
+    height: 4vw;
+}
+#playGameButton {
+    background-image: url("../assets/images/buttons/orangeButtonSmall.png");
+    color: #73340e;
 }
-
 /* code screen */
 #joinGameButtonAFterCodeInput, #playGameButton {
     width: auto;
     height: auto;
     margin: 1% 0 0 0;
+}
+
+#menuScreen {
+    transform: scale(0.9);
+}
+
+/* highscore */
+#highscoreList {
+    background-image: url("../assets/images/buttons/brownTable.png");
+    background-position: center;
+    color: #574741;
+    font-size: 1.5vw;
+    border-radius: 0;
+    margin-top: 1%;
+    background-size: cover;
+    margin-left: auto;
+    margin-right: auto;
+    width: 75%;
+}
+#highscoreList thead {
+    font-size: 1.5vw;
+}
+
+#highscoreList tbody th {
+    padding: 1%;
+    font-size: 1vw;
+}
+
+.highscoreReturnToMainButton {
+    left: 50%;
+    transform: translate(50%, 0);
+}
+
+#pages {
+    margin-left: auto;
+    margin-right: auto;
+    width: 75%;
+}
+
+#pages button {
+    background-image: url("../assets/images/buttons/orangeButtonSquare.png");
+    color: #73340e;
+    background-position: center;
+    background-size: cover;
+    border: none;
+    float: right;
```

Comment:
I think in the code here you will need to reverse printing of the buttons cause they are in the reverse order